shared/src/hooks/useNotifications.ts

```diff
@@ -77,7 +85,14 @@ const useNotifications = () => {
           type: "withdrawalReady",
           vault: vaultOption as VaultOptions,
           vaultVersion: "v2",
-          amount: vaultData.withdrawals.amount,
+          amount: vaultData.withdrawals.shares
+            .mul(priceHistory ? priceHistory.pricePerShare : BigNumber.from(0))
```

Comment:
If this displays `0`, it's better to change to a `-` because users will freak out if they have a balance and see zero.

Reply:
Nice catch, updated that so if price history does not exists it will not show at all